src/core/Akka.MultiNodeTestRunner/Program.cs

```diff
@@ -90,60 +106,22 @@ static void Main(string[] args)
                             continue;
                         }
 
-                        PublishRunnerMessage(string.Format("Starting test {0}", test.Value.First().MethodName));
-
-                        var processes = new List<Process>();
+                        if ( restrictToTestName && string.Compare(test.Key, specificationName, StringComparison.OrdinalIgnoreCase) != 0)
+                        {
+                            PublishRunnerMessage(string.Format("Skipping test {0}.{1}. Reason - Doesn't match multinode.spec-name.", test.Value.First().TestName, test.Value.First().MethodName));
+                            continue;
+                        }
 
-                        StartNewSpec(test.Value);
+                        PublishRunnerMessage(string.Format("Starting test {0}", test.Value.First().MethodName));
 
-                        foreach (var nodeTest in test.Value)
+                        if (!restrictToTestName)
```

Comment:
Do you want to pass in a global flag here to specify a strategy for running all tests? I.E. run ALL in AppDomains OR processes? Or is there a technical reason why this might be a bad idea - I.E. unpredictable results across multiple specs as a result of thread contention.